corehq/apps/app_manager/tests/data/test_util/advanced_module_case_management_app.json

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
Rather than adding these giant files is it not possible to create the app in code or use an existing one? See https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/test_schedule.py#L19